src/ExportedFunctions/Invoke-MSTerminalGif.ps1

```diff
@@ -0,0 +1,98 @@
+.EXAMPLE
+    PS C:\> Invoke-TerminalGif https://media.giphy.com/media/g9582DNuQppxC/giphy.gif
+    Triggers a gif in the current Windows Terminal
+.EXAMPLE
+    PS C:\> Invoke-TerminalGif (Search-Giphy -DirectUrl)
+    Triggers a gif in the current Windows Terminal
+.NOTES
+    Requires Windows Terminal Preview
+#>
+    [CmdletBinding()]
+    param (
+        #The URI of the GIF you want to display
+        [Parameter(Mandatory)][uri]$Uri,
+        #The name or GUID of the Windows Terminal Profile in which to play the Gif. If not specified, attempts to intelligently determine it from the executable name or if $ENV:WT_PROFILE was specified by adding -noexit -command \". {$env:WT_PROFILE='myprofilename'}\" to your terminal profile definition.
+        [String][Alias('GUID')]$Name,
+        #How to resize the background image in the window. Options are None, Fill, Uniform, and UniformToFill
+        [ValidateSet('none', 'fill', 'uniform', 'uniformToFill')][String]$StretchMode = 'uniformToFill',
+        #How transparent to make the background image. Default is 60% (.6)
+        [float]$BackgroundImageOpacity = 0.6,
+        #Specify this to use the Acrylic visual effect (semi-transparency)
+        [switch]$Acrylic,
+        #Maximum duration of the gif invocation in seconds
+        [int]$MaxDuration = 5
+    )
+
+    #Sanity Checks
+    if (-not $env:WT_SESSION) { throw "This only works in Windows Terminal currently. Please try running this command again inside a Windows Terminal powershell session." }
+    if ($PSEdition -eq 'Desktop' -and -not (Get-Command start-threadjob -erroraction silentlycontinue)) {
+        throw "This command requires the ThreadJob module on Windows Powershell 5.1. You can install it with the command Install-Module Threadjob -Scope CurrentUser"
+        return
+    }
+
+    #Pseudo Singleton to ensure only one prompt job is running at a time.
+    $InvokeTerminalGifJobName = 'InvokeTerminalGif'
+    $InvokeTerminalGifJob = Get-Job $InvokeTerminalGifJobName -Erroraction SilentlyContinue
+    if ($invokeTerminalGifJob) {
+        if ($invokeTerminalGifJob.state -notmatch 'Completed|Failed') {
+            Write-Warning "Terminal Gif Already Running"
+            return
+        } else {
+            Remove-Job $InvokeTerminalGifJob
+        }
+    }
+
+    $TerminalProfile = if ($Name -as [Guid]) {
+        Get-MSTerminalProfile -Guid $Name -ErrorAction stop
+    } else {
+        Get-MSTerminalProfile -Name $Name -ErrorAction stop
+    }
+
+    #Prepare arguments for the threadjob
+    $TerminalGifJobParams = @{ }
+    ('terminalprofile','uri', 'maxduration', 'stretchmode','acrylic','backgroundimageopacity').foreach{
+        $TerminalGifJobParams.$PSItem = (Get-Variable $PSItem).value
+    }
+
+    $TerminalGifJobParams.modulePath = (Get-Module msterminalsettings | % path) -replace 'psm1$','psd1'
```

Comment:
Can you update this to `(Get-Module msterminalsettings).path` (or ForEach-Object) so script analyzer won't complain?

Reply:
Sure, this was probably a holdover when I was just hacking away at a prototype.
Fixed https://github.com/gpduck/MSTerminalSettings/pull/45/commits/628720f02b45794e50e032c823fa4bc31e8a510d